automation_tools/__init__.py

```diff
@@ -918,7 +918,9 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         beta=distribution.endswith('beta'),
         host=host
     )
-    execute(update_packages, warn_only=True)
+
+    # Update the machine
+    update_packages(warn_only=True)
```

Comment:
The right fix would be something like, `execute(update_packages, warn_only=True, host=host)`.